vds_api_client/types.py

```diff
@@ -192,7 +193,7 @@ def ids_to_list(self):
         return [roi.id for roi in self._rois]
 
 
-class Roi(object):
+class Roi(Requester):
```

Comment:
I'm not sure if the `ROI` object should be a `Requester`. It could easily be that I misunderstand the architecture of the vds_api_client but it seems that it should just have an attribute that let's it access the Requester?

Reply:
I would like to implement api methods to the Roi objects that can call to the Roi endpoints themselves. Otherwise it'd need to be

```python
vds = VdsApiV2()
vds.update_roi(vds.rois[2930], description='new description')
```
since I cannot reference the vds object directly from the Roi object as they are unrelated. With the new setup, I can implement new features with methods that work directly on the Roi object like:

```python
roi = vds.rois[2930]
roi.geometry  # Extract geojson
roi.update(name='new_name', description='new description', display=True)
roi.add_label('label')
```
Do you mean I could use
```
Requester.post()
```
inside an `Roi` method without using inheritance?

Reply:
>Do you mean I could use

`>Requester.post()`

>inside an Roi method without using inheritance?

Yes that's what I meant. I think it would make more sense logically than making the `ROI` object a `Requester`. It also makes the code easier to understand I think.

Reply:
Ok, I changed Roi back to object now